src/main/java/core/basesyntax/FigureProducer.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureProducer {
+
+    public static final int COUNT_OF_FIGURE_IMPLEMENTATIONS = 5;
```

Comment:
```suggestion
    public static final int COUNT_OF_FIGURES = 5;
```
Figure does not have implamentation, cause it's a class